README.md

```diff
@@ -1,7 +1,151 @@
-
 # object-store-client
 
-This is a placeholder README.md for a new repository
+object-store-client is a Scala client to interact with the [object-store](https://github.com/hmrc/object-store) service. 
+The methods available on the client encapsulates the HTTP interaction with object-store service which makes it easier for the consuming service to interact with object-store service.
+
+The operations available via object-store-client are 
+- Put a new document
+- Download a document
+- Delete a document
+- List documents under a path
+
+Read more [here](https://github.com/hmrc/object-store/blob/master/README.md) about object-store.
+
+We also have a working example [hello-world-object-store](https://github.com/hmrc/hello-world-object-store) which shows in detail on how to use the object-store-client.
```

Comment:
" ~which shows in detail on how to~" 
"which shows, in detail, how to"
or 
"which shows detail on how to"